app/api/signIn.ts

```diff
@@ -0,0 +1,37 @@
+import { type SupabaseClient } from "@supabase/supabase-js";
+import { ErrorResponse } from "~/models/Error.model";
+import { validateLoginCredentials } from "~/util/auth/validate.server";
+
+type SignInResponse = {
+  accessToken?: string;
+  refreshToken?: string;
+  error?: ErrorResponse;
+};
+
+export async function signIn(
+  email: string,
```

Comment:
@Pinwheeler What is your opinion on each argument having it's own line? Prettier does this when the argument exceeds the default printWidth. I'm curious if I should just make the printWidth larger of if this is fine.

Reply:
No real preference here. If you want to try it the other way, I'm game